traffic_ops/testing/api/v3/deliveryservices_test.go

```diff
@@ -69,6 +71,21 @@ func TestDeliveryServices(t *testing.T) {
 	})
 }
 
+func GetTestDeliveryServicesV31(t *testing.T) {
```

Comment:
We shouldn't need new minor-version specific tests if we follow my first comment above. We could just check this new field in the existing `GetTestDeliveryServices` test (similar for update).